Gemfile

```diff
@@ -43,7 +43,7 @@ gem "jquery-ui-rails"
 gem "simple_form", '~> 3.1.0'
 gem 'nokogiri'
 gem 'equivalent-xml'
-gem 'solr_wrapper'
+gem 'solr_wrapper', '0.7.3'
```

Comment:
Is there a particular dependency on 0.7.3? Or can we keep current (looks like 0.9.0 has been released).